pysrc/qulacs/__init__.pyi

```diff
@@ -449,38 +449,137 @@ class QuantumCircuit:
+                \\begin{pmatrix}
+                \cos(\\frac{\\theta}{2})  & i\sin(\\frac{\\theta}{2}) \\\\
+                i\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
+                \end{pmatrix}
         """
     def add_RY_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-Y rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_Y(\\theta) = \exp(i\\frac{\\theta}{2} Y) =
+                \\begin{pmatrix}
+                \cos(\\frac{\\theta}{2})  & \sin(\\frac{\\theta}{2}) \\\\
+                -\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
+                \end{pmatrix}
         """
     def add_RZ_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-Z rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_Z(\\theta) = \exp(i\\frac{\\theta}{2} Z) =
+                \\begin{pmatrix}
+                e^{i\\frac{\\theta}{2}} & 0 \\\\
+                0 & e^{-i\\frac{\\theta}{2}}
+                \end{pmatrix}
         """
     def add_RotInvX_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-X rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_X(\\theta) = \exp(i\\frac{\\theta}{2} X) =
+                \\begin{pmatrix}
+                \cos(\\frac{\\theta}{2})  & i\sin(\\frac{\\theta}{2}) \\\\
+                i\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
+                \end{pmatrix}
         """
     def add_RotInvY_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-Y rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_Y(\\theta) = \exp(i\\frac{\\theta}{2} Y) =
+                \\begin{pmatrix}
+                \cos(\\frac{\\theta}{2})  & \sin(\\frac{\\theta}{2}) \\\\
+                -\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
+                \end{pmatrix}
         """
     def add_RotInvZ_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-Z rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_Z(\\theta) = \exp(i\\frac{\\theta}{2} Z) =
+                \\begin{pmatrix}
+                e^{i\\frac{\\theta}{2}} & 0 \\\\
+                0 & e^{-i\\frac{\\theta}{2}}
+                \end{pmatrix}
         """
     def add_RotX_gate(self, index: int, angle: float) -> None:
         """
         Add Pauli-X rotation gate
+
+        Notes
+        -----
+        Matrix Representation
+
+        .. math::
+            R_X(-\\theta) = \exp(-i\\frac{\\theta}{2} X) =
+                \\begin{pmatrix}
+                \cos(\\frac{\\theta}{2})  & -i\sin(\\frac{\\theta}{2}) \\\\
+                -i\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
+                \end{pmatrix}
```

Comment:
Matrix representation of `RotX` should be like
```suggestion
        .. math::
            RotX(\\theta) = \exp(-i\\frac{\\theta}{2} X) =
                \\begin{pmatrix}
                \cos(\\frac{\\theta}{2})  & -i\sin(\\frac{\\theta}{2}) \\\\
                -i\sin(\\frac{\\theta}{2}) & \cos(\\frac{\\theta}{2})
                \end{pmatrix}
```

- `RotX`, not `R_X`
- Negation in the argument is not needed

Reply:
changed to `RotX`